packages/jest-fake-chalk/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "jest-fake-chalk",
```

Comment:
Would it make sense to have this living outside the packages folder as it really isn't a package, but a stub used only in the browser build?

Reply:
then it's a bit messy plugging in dependencies, but I have no feelings on the matter

Reply:
👍